src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -33,46 +32,46 @@ android {
 ```
 
 The Flutter engine has an `x86` and `x86_64` version. When using an emulator
-in debug Just-In-Time (JIT) mode, the Flutter module will still run correctly.
+in debug Just-In-Time (JIT) mode, the Flutter module still runs correctly.
 
 TODO(blasten): revise whether this is right.
 {{site.alert.end}}
 
 ## Using Android Studio
 
-The Android Studio IDE offers a convenient way of integrating your Flutter
+The Android Studio IDE is a convenient way of integrating your Flutter
 module automatically. With Android Studio, you can co-edit both your Android
 code and your Flutter code in the same project. You can also continue to use
 your normal IntelliJ Flutter plugin functionalities such as Dart code completion,
-hot reload, widget inspector etc.
+hot reload, and widget inspector.
```

Comment:
perhaps code completion, hot reload, widget inspector, and more? Since this is a very incomplete list.

Reply:
That’s fine. Per Google style, don’t use Latin terms. That’s what I was trying to avoid doing.

Reply:
Understood.